### PR TITLE
New footer design

### PR DESCRIPTION
**Pull request by megaslug**
screen shot 2016-12-21 at 18 06 04

### CODE REVIEW

**Comment by clippy** [comment]
**I have manually checked the visuals in:**

- [ ] Chrome
- [ ] IE11
- [ ] Firefox
- [ ] Safari
- [ ] iPad

File: app/components/footer/footer.js [hunk 1]
```diff
@@ -45,151 +46,159 @@ class Footer extends React.Component {
...
-                        </li>
-                      );
-                    })
-                  }
+      <div>
+        <Translation locales='en'>
+          <FooterEn/>
+        </Translation>
+
+        <Translation locales={availableLocales} exclude={['en-GB', 'en-EU', 'en-IE', 'en-SE']}>
```
Comment (Ken-Lo) [inline app/components/footer/footer.js]: @megaslug Any reason why each locale is specifically listed specifically here, but above it's just `en`?

File: app/components/footer/footer.en.js [hunk 2]
```diff
@@ -0,0 +1,322 @@
...
+                <IfLinkExists to='new_to_direct_debit' tagName='li'>
+                  <Link to='new_to_direct_debit' pointer='new_to_direct_debit.nav_title' className='page-footer__link'/>
+                </IfLinkExists>
+                <IfLinkExists to='existing_direct_debit_user' tagName='li'>
+                  <Link to='existing_direct_debit_user' pointer='existing_direct_debit_user.nav_title' className='page-footer__link'/>
+                </IfLinkExists>
+                <IfLinkExists to='developer_features' tagName='li'>
+                  <p className='page-footer__link'>
+                    <Link to='developer_features'>
+                      Learn about our API
```
Comment (Ken-Lo) [inline app/components/footer/footer.en.js]: Good to add this as a message as most other links use one?

File: app/components/footer/footer.js (hunk 1 above)
Comment (megaslug) [inline app/components/footer/footer.js]: @Ken-Lo the new footer is only applicable to EN and not other locales

File: app/components/footer/footer.en.js (hunk 2 above)
Comment (megaslug) [inline app/components/footer/footer.en.js]: will add this one in. thanks.

**Pull request merged**